src/openforms/setup.py

```diff
@@ -140,3 +141,12 @@ def mute_deprecation_warnings():
     """
     if not sys.warnoptions:
         warnings.simplefilter("ignore", DeprecationWarning, append=False)
+
+
+def monkeypatch_json_logic():
```

Comment:
can you please add a TODO that we need to apply this fix to our fork of json-logic-py?